src/data/gods/hera.ts

```diff
@@ -136,11 +187,137 @@ const uncommonGrace: Boon = {
 const dyingWish: Boon = {
   name: "Dying Wish",
   type: OTHER,
+  element: EARTH,
+  info: (value) =>
+    `Whenever [hitch]-afflicted foes are slain, damage all other [hitch] afflicted foes for ${value}`,
+  values: {
+    [COMMON]: "40",
+  },
+  prerequisites: [[attack, special, dash, nastyComeback]],
+};
+
+export const braveFace: Boon = {
+  name: "Brave Face",
+  type: OTHER,
+  element: EARTH,
+  info: (value) =>
+    `Automatically use ${value} magic to resist up to 50% of any damage`,
+  values: {
+    [LEGENDARY]: { 1: 5 },
+  },
+  prerequisites: [
+    [attack, special],
+    [bornGain, keenIntuition],
+    [hereditaryBane, dyingWish],
+  ],
+};
+
+const funeralPyre: Boon = {
+  name: "Funeral Pyre",
+  type: OTHER,
+  element: COSMIC,
+  info: (value) =>
+    `While you Channel your [omega] moves, repeatedly inflict ${value} [scorch] on nearby foes`,
+  values: {
+    [DUO]: { 1: 90 },
+  },
+  prerequisites: [
+    [attack, special, cast, bornGain],
+    [flameStrike, flameFlourish, smolderRing, hearthGain],
+  ],
+};
+
+const spitefulStrength: Boon = {
+  name: "Spiteful Strength",
+  type: OTHER,
+  element: COSMIC,
+  info: (value) =>
+    `Your Attacks and Specials deal ${value} more damage while not empowered by Boons`,
+  values: {
+    [DUO]: "200%",
+  },
+  prerequisites: [
+    [braveFace, nastyComeback, keenIntuition, bornGain],
+    [
+      trustyShield,
+      mintCondition,
+      heavyMetal,
+      toughTrade,
+      uncannyFortitude,
+      fixedGain,
+    ],
+  ],
+};
+
+const cherishedHeirloom: Boon = {
+  name: "Cherished Heirloom",
+  type: OTHER,
+  info: (value) =>
+    `Most other Keepsakes you equip are ${value} ranks strong this night (if possible)`,
+  values: {
+    [DUO]: { 1: 1 },
```

Comment:
Some of the duos have a values object like this, but I believe that Duos and Legendaries cannot level up.

Reply:
You are correct, although with how under-tuned some of the Duos are I'm not positive that's their intent for this game. Using this format isn't *technically* wrong because it is there level 1 value... there just is no level 2. The bigger reason is just so that its consistent with the other boons though